diff --git a/PKGBUILD b/PKGBUILD
index e04f450..1c73ec1 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -9,12 +9,15 @@ url="https://www.fooyin.org/"
 _ghurl="https://github.com/ludouzi/fooyin"
 license=('GPL-3.0-only')
 depends=(
+    'icu'
+    'glibc'
+    'libstdc++'
+    'libgcc'
     'qt6-base'
     'alsa-lib'
     'taglib'
     'ffmpeg'
     'kdsingleapplication'
-    'libvgm-git'
     'qcoro'
 )
 makedepends=(
@@ -30,7 +33,6 @@ makedepends=(
     'libarchive'
     'libgme'
     'soundtouch'
-    'git'
 )
 optdepends=(
     'sdl2: For the SDL2 audio output plugin'
@@ -40,7 +42,8 @@ optdepends=(
     'libsndfile: For the GME audio input plugin'
     'libarchive: For the libarchive archive plugin'
     'libebur128: For the ReplayGain scanner plugin'
-    'soundtouch: Enable building a dsp plugin based on soundtouch.'
+    'libsoxr: For the SoX-based DSP plugin'
+    'soundtouch: For the SoundTouch-based DSP plugin'
 )
 source=(
     "${pkgname}-${pkgver}.tar.gz::${_ghurl}/archive/refs/tags/v${pkgver}.tar.gz"
@@ -57,6 +60,8 @@ build() {
     cmake --build build
 }
 package() {
+    depends+=('hicolor-icon-theme')
+
     cd "${srcdir}/${pkgname}-${pkgver}"
     DESTDIR="${pkgdir}" cmake --install build
     install -Dm644 "${srcdir}/${pkgname}-${pkgver}/COPYING" "${pkgdir}/usr/share/licenses/fooyin/LICENSE"
