diff --git a/PKGBUILD b/PKGBUILD
index c024388..7cdd4f0 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -47,7 +47,6 @@ source=(
 sha256sums=('e702389488e19c4c48b1b62bf1b2adf263b818138e3b232a39259057cbcec9c2')
 build() {
     cd "${srcdir}/${pkgname}-${pkgver}"
-    find src -type f -print0 | xargs -0 sed -i 's/opt.backgroundBrush = {};/opt.backgroundBrush = Qt::NoBrush;/g'
     cmake -S . -B build -G Ninja \
         -DCMAKE_INSTALL_PREFIX=/usr \
         -DBUILD_PCH=ON \
