diff --git a/PKGBUILD b/PKGBUILD
index 0bc236a..5327ad2 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,20 +1,23 @@
 # Maintainer: Morgan <morganamilo@archlinux.org>
 pkgname=paru
 pkgver=2.1.0
-pkgrel=3
+pkgrel=4
 pkgdesc='Feature packed AUR helper'
 url='https://github.com/morganamilo/paru'
-source=("$pkgname-$pkgver.tar.gz::https://github.com/Morganamilo/paru/archive/v$pkgver.tar.gz")
+source=("$pkgname-$pkgver.tar.gz::https://github.com/Morganamilo/paru/archive/v$pkgver.tar.gz"
+        "Remove-bind-ro-for-makepkg.conf.d.diff::https://github.com/Morganamilo/paru/pull/1407.diff")
 backup=("etc/paru.conf")
 arch=('i686' 'pentium4' 'x86_64' 'arm' 'armv7h' 'armv6h' 'aarch64')
 license=('GPL-3.0-or-later')
 makedepends=('cargo' 'clang')
 depends=('git' 'pacman' 'libalpm.so>=14')
 optdepends=('bat: colored pkgbuild printing' 'devtools: build in chroot and downloading pkgbuilds')
-sha256sums=('eea4dbb524db765d5316f540f9ee670c0bf81aae4827b5417eebb4c9b5651727')
+sha256sums=('eea4dbb524db765d5316f540f9ee670c0bf81aae4827b5417eebb4c9b5651727'
+            '18d1481a138a8d7b06f7ccbbf5e54b6b8604d236cb85bc9b2f7d26ac1c33ccc6')
 
 prepare() {
   cd "$pkgname-$pkgver"
+  patch -Np1 -i ../Remove-bind-ro-for-makepkg.conf.d.diff
   cargo update alpm alpm-utils
   cargo fetch --locked --target "$(rustc -vV | sed -n 's|host: ||p')"
 }
