diff --git a/Makefile b/Makefile
index 03496c20..11ce6161 100644
--- a/Makefile
+++ b/Makefile
@@ -85,7 +85,7 @@ install-go-tools:
 	go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.34.2
 
 ## Build:
-build: protogen-go install-go-tools ## Build the project
+build: protogen-go ## Build the project
 	$(info ${GREEN}I local-ai build info:${RESET})
 	$(info ${GREEN}I BUILD_TYPE: ${YELLOW}$(BUILD_TYPE)${RESET})
 	$(info ${GREEN}I GO_TAGS: ${YELLOW}$(GO_TAGS)${RESET})
@@ -277,9 +277,9 @@ protoc:
 	unzip -j -d $(CURDIR) protoc.zip bin/protoc && rm protoc.zip
 
 .PHONY: protogen-go
-protogen-go: protoc install-go-tools
+protogen-go:
 	mkdir -p pkg/grpc/proto
-	./protoc --experimental_allow_proto3_optional -Ibackend/ --go_out=pkg/grpc/proto/ --go_opt=paths=source_relative --go-grpc_out=pkg/grpc/proto/ --go-grpc_opt=paths=source_relative \
+	protoc --experimental_allow_proto3_optional -Ibackend/ --go_out=pkg/grpc/proto/ --go_opt=paths=source_relative --go-grpc_out=pkg/grpc/proto/ --go-grpc_opt=paths=source_relative \
     backend/backend.proto
 
 .PHONY: protogen-go-clean
