diff --git a/backend/go/local-store/Makefile b/backend/go/local-store/Makefile
index b342f3f..061b3b5 100644
--- a/backend/go/local-store/Makefile
+++ b/backend/go/local-store/Makefile
@@ -1,7 +1,7 @@
 GOCMD=go
 
 local-store:
-	CGO_ENABLED=0 $(GOCMD) build -ldflags "$(LD_FLAGS)" -tags "$(GO_TAGS)" -o local-store ./
+	CGO_ENABLED=0 $(GOCMD) build -ldflags "$(LD_FLAGS) -bindnow" -tags "$(GO_TAGS)" -o local-store ./
 
 package:
 	bash package.sh
