diff --git a/backend/go/silero-vad/Makefile b/backend/go/silero-vad/Makefile
index 3b3c31e..84f476d 100644
--- a/backend/go/silero-vad/Makefile
+++ b/backend/go/silero-vad/Makefile
@@ -37,9 +37,9 @@ else
 	mv backend-assets/lib/libonnxruntime.so.$(ONNX_VERSION) backend-assets/lib/libonnxruntime.so.1
 endif
 
-silero-vad: backend-assets/lib/libonnxruntime.so.1
-	CGO_LDFLAGS="$(CGO_LDFLAGS)" CPATH="$(CPATH):$(CURRENT_DIR)/sources/onnxruntime/include/" LIBRARY_PATH=$(CURRENT_DIR)/backend-assets/lib \
-	$(GOCMD) build -ldflags "$(LD_FLAGS)" -tags "$(GO_TAGS)" -o silero-vad ./
+silero-vad:
+	CGO_LDFLAGS="$(CGO_LDFLAGS)" CGO_CFLAGS="$(CGO_CFLAGS) $(shell pkg-config --cflags libonnxruntime)" CGO_LDFLAGS="$(CGO_LDFLAGS) $(shell pkg-config --libs libonnxruntime)" \
+	$(GOCMD) build -ldflags "$(LD_FLAGS) -compressdwarf=false -linkmode external -bindnow" -tags "$(GO_TAGS)" -o silero-vad ./
 
 package:
 	bash package.sh
