diff --git a/backend/go/voxtral/Makefile b/backend/go/voxtral/Makefile
index 5385a9d..8966aa9 100644
--- a/backend/go/voxtral/Makefile
+++ b/backend/go/voxtral/Makefile
@@ -62,7 +62,7 @@ sources/voxtral.c:
 	git submodule update --init --recursive --depth 1 --single-branch
 
 voxtral: main.go govoxtral.go $(VARIANT_TARGETS)
-	CGO_ENABLED=0 $(GOCMD) build -tags "$(GO_TAGS)" -o voxtral ./
+	CGO_ENABLED=0 $(GOCMD) build -ldflags "-bindnow" -tags "$(GO_TAGS)" -o voxtral ./
 
 package: voxtral
 	bash package.sh
