diff --git a/src/test.cpp b/src/test.cpp
index b7619a1..c6d1e46 100644
--- a/src/test.cpp
+++ b/src/test.cpp
@@ -2,6 +2,7 @@
 #include <map>
 #include <sstream>
 #include <vector>
+#include <cstdint>
 
 #include <espeak-ng/speak_lib.h>
 
@@ -119,7 +120,7 @@ int main(int argc, char *argv[]) {
   if (phonemes[0] != expectedPhonemes) {
     std::cerr << "Весе́лка: ";
     for (auto phoneme : phonemes[0]) {
-      std::cerr << phoneme << " ";
+      std::cerr << static_cast<uint32_t>(phoneme) << " ";
     }
 
     std::cerr << std::endl;
