diff --git a/backend/go/cloud-proxy/Makefile b/backend/go/cloud-proxy/Makefile
index 7900905..a0831ad 100644
--- a/backend/go/cloud-proxy/Makefile
+++ b/backend/go/cloud-proxy/Makefile
@@ -1,7 +1,7 @@
 GOCMD=go
 
 cloud-proxy:
-	CGO_ENABLED=0 $(GOCMD) build -ldflags "$(LD_FLAGS)" -tags "$(GO_TAGS)" -o cloud-proxy ./
+	CGO_ENABLED=0 $(GOCMD) build -ldflags "$(LD_FLAGS) -bindnow" -tags "$(GO_TAGS)" -o cloud-proxy ./
 
 package:
 	bash package.sh
