diff --git a/backend/go/ced/Makefile b/backend/go/ced/Makefile
index 632c0e2..f0151fd 100644
--- a/backend/go/ced/Makefile
+++ b/backend/go/ced/Makefile
@@ -60,7 +60,7 @@ libced.so: sources/ced.cpp
 	cp -fv sources/ced.cpp/include/ced_capi.h ./
 
 ced-grpc: libced.so main.go goced.go
-	CGO_ENABLED=0 $(GOCMD) build -tags "$(GO_TAGS)" -o ced-grpc .
+	CGO_ENABLED=0 $(GOCMD) build -ldflags "-bindnow" -tags "$(GO_TAGS)" -o ced-grpc .
 
 package: ced-grpc
 	bash package.sh
