diff --git a/PKGBUILD b/PKGBUILD
index 5adf83a..239a117 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -11,7 +11,7 @@ license=('MIT')
 optdepends=("ungoogled-chromium")
 source=("${pkgname}-${pkgver}.crx::${url}/releases/download/v${pkgver}/Chromium.Web.Store.crx")
 noextract=("${pkgname}-${pkgver}.crx")
-sha256sums=('80a01caef9f12e1f6032b693157e3346c52e6b8388fa56a4e75829208dbd77c8')
+sha256sums=('e593bf206d40a7b947d87a84c208f3967e0e8b9a2ac4d278c07cb1a21f996532')
 install=chromium-extension-web-store.install
 
 prepare() {
