diff --git a/PKGBUILD b/PKGBUILD
index 6a386ab..ab9ed11 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: morpheusthewhite <zffromGerace@hotmail.it>
 
 pkgname=spicetify-themes-git
-pkgver=r404.4f10f4f
+pkgver=r496.ca42642
 pkgrel=1
 pkgdesc="A community-driven collection of themes for spicetify"
 arch=('any')
@@ -12,12 +12,8 @@ depends=('spicetify-cli')
 makedepends=('git')
 provides=("${pkgname%-git}")
 conflicts=("${pkgname%-git}")
-replaces=()
-backup=()
-options=()
 install=spicetify-themes-git.install
 source=("$pkgname"::"git+https://github.com/spicetify/spicetify-themes")
-noextract=()
 md5sums=('SKIP')
 
 pkgver() {
@@ -29,8 +25,9 @@ pkgver() {
 }
 
 package() {
-    mkdir -p "${pkgdir}"/usr/share/spicetify-cli/Extensions
-
-    cp -R "$srcdir/${pkgname}" "${pkgdir}"/usr/share/spicetify-cli/Themes
-    rm -r "${pkgdir}"/usr/share/spicetify-cli/Themes/README.md "${pkgdir}"/usr/share/spicetify-cli/Themes/LICENSE
+    mkdir -p "${pkgdir}"/opt/spicetify-cli
+    local themes="${pkgdir}"/opt/spicetify-cli/Themes
+    cp -R "$srcdir/${pkgname}" "$themes"
+    rm -rf "$themes"/{*.md,LICENSE,.git*,_Extra,manifest.json}
+    install -Dm644 "$srcdir/${pkgname}/LICENSE" "${pkgdir}/usr/share/licenses/$pkgname/LICENSE"
 }
