diff --git a/PKGBUILD b/PKGBUILD
index 29e24d8..e464307 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,7 +8,7 @@ pkgdesc="Qcm backend with Rust"
 url="https://github.com/hypengw/QcmBackend"
 arch=('x86_64' 'aarch64' 'riscv64')
 license=('MPL-2.0')
-depends=('protobuf' 'openssl' 'libgcc' 'sqlite')
+depends=('protobuf' 'openssl' 'libgcc' 'sqlite' 'lua')
 makedepends=('git' 'cargo')
 conflicts=("${_pkgname}")
 provides=("${_pkgname}")
