diff --git a/PKGBUILD b/PKGBUILD
index 1ebbff8..69526f1 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -60,7 +60,7 @@ _package() {
 				'linux-firmware: firmware images needed for some devices'
 				'scx-scheds: to use sched-ext schedulers'
 				'wireless-regdb: to set the correct wireless channels of your country')
-	provides=(linux-pf KSMBD-MODULE NTSYNC-MODULE OVPN-MODULE VIRTUALBOX-GUEST-MODULES WIREGUARD-MODULE)
+	provides=(KSMBD-MODULE NTSYNC-MODULE OVPN-MODULE VIRTUALBOX-GUEST-MODULES WIREGUARD-MODULE)
 	replaces=(virtualbox-guest-modules-arch wireguard-arch)
 
 	cd linux
@@ -85,7 +85,7 @@ _package() {
 _package-headers() {
 	pkgdesc="Headers and scripts for building modules for the ${pkgdesc}"
 	depends=(pahole)
-	provides=(LINUX-HEADERS linux-pf-headers)
+	provides=(LINUX-HEADERS)
 
 	cd linux
 	local builddir="${pkgdir}"/usr/lib/modules/$(<version)/build
