diff --git a/PKGBUILD b/PKGBUILD
index e5c43d0..ead2e01 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -26,5 +26,5 @@ package() {
     install -Dm644 "LICENSE" "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
 
     cd "packages/ui/.output/public"
-    find . -type f -exec install -Dm644 {} "${pkgdir}/usr/share/${pkgname}/"{} \;
+    find . -type f -exec install -Dm644 {} "${pkgdir}/usr/share/webapps/${pkgname}/"{} \;
 }
